api/src/db/migrations/migrations.ts

```diff
@@ -298,6 +299,7 @@ export const Migrations: MigrationFunction[] = [
   migrate_v144_to_v145,
   migrate_v145_to_v146,
   migrate_v146_to_v147,
+  migrate_v147_to_v148,
 ];
 
 export { generatev144UserData as CURRENT_GENERATOR } from "@db/migrations/v144_add_raffle_bingo_games";
```

Comment:
IF we add a migration, then we need to advance this line.

It looks like that's been missed a couple times. There's a test that should be checking this, but for some reason that didn't trigger.